wgpu/src/lib.rs

```diff
@@ -495,7 +497,8 @@ trait Context: Debug + Send + Sized + Sync {
     fn queue_on_submitted_work_done(
         &self,
         queue: &Self::QueueId,
-    ) -> Self::OnSubmittedWorkDoneFuture;
+        callback: Box<dyn FnOnce() + Send + 'static>,
```

Comment:
Basically to prevent needing to make the function generic if we don't have to, there's a comment on the other function explaining why we use `impl`, but I should probably add a comment here.

Reply:
Added.